Tasks/Salanenka/Task3/Task3.py

```diff
@@ -0,0 +1,74 @@
+from datetime import datetime
+
+while True:
+    type = input("Введите 1 если хотите вывести текущее время и 2, если хотите ввести время вручную \n")
+    if type != '1' and type != '2':
+        print ('Ошибка.')
+        continue
+    elif type == '1':
+        time_input = datetime.now()
+        print ('Текущее время:', time_input.strftime("%H:%M"))
+        hour = time_input.hour
+        min = time_input.minute
+    else:    
+        time_input = input("Введите время в формате: hh:mm \n")
+        if len(time_input) != 5:
+            print("Введено неверное количество символов")
+            continue
+        if time_input[2] != ":":
+            print("Ошибка. Не хватает символа ':'")
+            continue
+        time_split = (time_input.split(':'))
+        hour = time_split[0]
+        min = time_split[1]
+        if not (hour.isdigit() and min.isdigit()):
+            print("Ошибка. Должны быть введены только цифры")
+            continue
+        hour, min = int(hour), int(min)
+        if hour > 24:
+            print("Ошибка. Часы должны быть в диапазоне от 00 до 24")
+            continue
+        if min > 59:
+            print("Ошибка. Минуты должны быть в диапазоне от 00 до 59")
+            continue
+        print(f'Вы ввели: {time_input}')
+    break
+min1 = min-min%10
+min2 = min%10
+if hour in range (13,24):
+    hour -= 12
+d1 = {1:'один', 2:'два', 3:'три', 4:'четыре', 5:'пять', 6:'шесть', 7:'семь', 8:'восемь', 9:'девять',10:'десять', 11:'одиннадцать', 12:'Двенадцать'}
+d2 = {12:'первого', 1:'второго', 2:'третьего', 3:'четвертого', 4:'пятого', 5:'шестого', 6:'седьмого', 7:'восьмого', 8:'девятого', 9:'десятого', 10:'одиннадцатого', 11:'двенадцатого'}
+d3= {1:'одна', 2:'две', 3:'три', 4:'четыре', 5:'пять', 6:'шесть', 7:'семь', 8:'восемь', 9:'девять', 10:'десять', 11:'одиннадцать', 12:'двенадцать', 13:'тринадцать', 14:'четырнадцать', 15:'пятнадцать', 16:'шестнадцать', 17:'семнадцать', 18:'восемнадцать', 19:'девятнадцать', 20:'двадцать', 30: 'тридцать', 40: 'сорок'}
+d4 = {58: 'двух', 57: 'трех', 56: 'четырех', 55: 'пяти', 54: 'шести', 53: 'семи', 52: 'восьми', 51: 'девяти', 50: 'десяти', 49: 'одиннадцати', 48: 'двенадцати', 47: 'тринадцати', 46: 'четырнадцати', 45: 'пятнадцати'}
```

Comment:
there are several duplicated values in the structure, not a big mistake, but it would be nice not to have it that way